vscode-extension/media/main.html

```diff
@@ -23,18 +23,22 @@ <h1>Error Central</h1>
     <!-- ec data here -->
   </div>
 
+  <h2> Stack Overflow Questions </h2>
+  <div id="stackex-questions">
```

Comment:
Minor point, but on past teams we've not done shortened names like 'stackex'. The problem is that with more people on the team, over time every possible shorting starts to show up in the code, thus leading to hard-to notice bugs. E.g. by next year we could have names 'stackex', 'stkex', 'stackX', 'se', 'stckx', 'stckxng',  etc... The policy that has seemed to work best is to limit to full name 'stackexchange' or full abbreviation 'se'.